src/node/txdownload_impl.cpp

```diff
@@ -436,4 +436,13 @@ CTransactionRef TxDownloadImpl::GetTxToReconsider(NodeId nodeid)
     return m_orphanage.GetTxToReconsider(nodeid);
 }
 
+void TxDownloadImpl::CheckIsEmpty(NodeId nodeid)
```

Comment:
CheckRequestsEmpty?

Reply:
`CheckIsEmpty()` seems a bit more future-proof. We'll check
- no requests for this peer
- no pending orphan resolutions for this peer
- no orphans for this peer
etc.